src/components/molecules/LoginComponent/LoginSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Header from "../Header";
+import Image from "next/image";
+import BannerImage from "@/../../public/images/banner.png";
+import CopyrightInfo from "@/components/atoms/CopyrightInfo";
+import Link from "next/link";
+import styles from "./LoginSection.module.scss";
+import { MdOutlineAccessTimeFilled } from "react-icons/md";
```

Comment:
I think there's no need to install react-icons because we have a reusable InfoCard component in atoms. Let's use as many of our built components as possible.